themes/dfs_base/scss/base/_font-settings.scss

```diff
@@ -0,0 +1,68 @@
+//fonts used for fin
```

Comment:
@briancwald How about keeping the icon fonts and dropping the others? I can see us using iconicons and fontawesome icons in future demos. I also plan to do code review today and clean up the comments which should remove instances of "fin". Sorry about that!